compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/BallerinaParser.java

```diff
@@ -1949,15 +1948,27 @@ private STNode parseParameterRhs(SyntaxKind tokenKind) {
      */
     private STNode parseParameter(SyntaxKind prevParamKind, boolean isParamNameOptional) {
         STNode annots;
+        STNode inclusionSymbol;
         STToken nextToken = peek();
         switch (nextToken.kind) {
             case AT_TOKEN:
                 annots = parseOptionalAnnotations();
+                if (peek().kind == SyntaxKind.ASTERISK_TOKEN) {
+                    inclusionSymbol = parseAsteriskToken();
```

Comment:
We can directly use `consume()` here

Reply:
Fixed with https://github.com/ballerina-platform/ballerina-lang/pull/27662